### PR TITLE
Newsletter for April 2022

### PR DESCRIPTION
**Pull request by milancurcic**
Here's the draft for the April newsletter.

Outstanding are compiler updates:

* [x] Flang by @AlexisPerry 
* [ ] LFortran by @certik 

Let's aim to merge this by Wednesday, April 6.

### CODE REVIEW

**Comment by AlexisPerry** [comment]
Please see PR https://github.com/milancurcic/fortran-lang.github.io/pull/6 for Flang updates.  Thanks!

File: _posts/2022-04-01-Fortran-Newsletter-April-2022.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
+---
+layout: post
+title: "Fortran newsletter: April 2022"
+category: newsletter
+author: Milan Curcic
```
Comment (milancurcic) [inline _posts/2022-04-01-Fortran-Newsletter-April-2022.md:5]:
```suggestion
author: Milan Curcic, Alexis Perry-Holby
```

**Comment by milancurcic** [comment]
@certik any chance for an LFortran update today?

**Comment by milancurcic** [comment]
#build_preview

**Comment by milancurcic** [comment]
WIll go ahead and merge in interest of time.

@certik please open a separate PR if you'd like to add LFortran updates.

**Pull request merged**